content/en/docs/concepts/secure-deployment.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Securely deploying Notation"
+description: "Best practices for securely deploying Notation"
+type: docs
+weight: 2
+---
+
+A deployment of Notation, specifically the `notation` CLI, is used for signing and verifying container images. The `notation` CLI is usually deployed on a development computer or in a CI/CD pipeline, such as GitHub Actions.
+
+A deployment has several sensitive components:
+
+- [Trust policies]({{< ref "/docs/how-to/manage-trust-policy" >}})
+- [Trust stores and certificates]({{< ref "/docs/concepts/directory-structure" >}})
+- Plugins
+
+It is strongly recommended that you follow the best practices in this document to secure your deployment of Notation.
```

Comment:
i think that the risks of not following secure best practices can be called out as part of this initial section. example:

> You are responsible for securing your deployment of Notation and following the best practices outlined in this document. Failure to do so can result in malicious plugins being installed, trust policies being modified to allow verification compromised or untrusted artifacts, trust stores being modified to have untrusted or compromised certificates, or signing keys being compromised.

Reply:
Good call, relocated content to this section